Document/SUMMARY.md

```diff
@@ -29,7 +29,7 @@
 - [Android Network Communication](0x05g-Testing-Network-Communication.md)
 - [Android Platform APIs](0x05h-Testing-Platform-Interaction.md)
 - [Code Quality and Build Settings for Android Apps](0x05i-Testing-Code-Quality-and-Build-Settings.md)
-- [Tampering and Reverse Engineering on Android](0x05c-Reverse-Engineering-and-Tampering.md)
+- [Reverse Engineering on Android](../techniques/android/MASTG-TECH-0013.md)
```

Comment:
This was a chapter previously, not the technique itself. We can remove it since that chapter does not exist anymore. All techniques are merged in the PDF into 0x05b/0x06b.
```suggestion
```